### PR TITLE
Update tools for the changed decoder API

### PR DESCRIPTION
**Pull request by sdroege**
And some other minor commits.

### CODE REVIEW

File: dav1d-sys/build.rs [hunk 1]
```diff
@@ -34,11 +34,21 @@ mod build {
         let release_path = source.join(release_dir);
 
         if !Path::new(&source.join(".git")).exists() {
-            runner!("git", "clone", "--depth", "1", REPO, &source);
+            runner!("git", "clone", REPO, &source);
```
Comment (lu-zero) [inline dav1d-sys/build.rs:37]:
```suggestion
            runner!("git", "clone", "--depth", "1", "-b", "0.9.2", REPO, &source);
```

File: dav1d-sys/build.rs [hunk 2]
```diff
@@ -34,11 +34,21 @@ mod build {
...
         } else {
             runner!("git", "-C", source.to_str().unwrap(), "pull");
         }
 
+        runner!("git", "-C", source.to_str().unwrap(), "checkout", "0.9.2");
```
Comment (lu-zero) [inline dav1d-sys/build.rs:42]: it needs a 
```
git fetch --depth 1 origin <newtag> && git checkout FETCH_HEAD
```

and can be folded in the branch above.

**Review by lu-zero — CHANGES_REQUESTED** [review]

File: dav1d-sys/build.rs (hunk 2 above)
Comment (sdroege) [reply dav1d-sys/build.rs:42]: Ah via `fetch --depth 1` that should actually work. That's what I was missing, thanks!

**Review by lu-zero — APPROVED** [review]
Thank you!

**Pull request merged**